driver/CMakeLists.txt

```diff
@@ -137,12 +117,16 @@ endif()
 # http://public.kitware.com/Bug/view.php?id=8438
 if(BUILD_DRIVER)
 	add_custom_target(driver ALL
+		COMMAND "${CMAKE_COMMAND}" -E copy_if_different ${CMAKE_CURRENT_BINARY_DIR}/src/Makefile ${CMAKE_CURRENT_SOURCE_DIR}/Makefile
+		COMMAND "${CMAKE_COMMAND}" -E copy_if_different ${CMAKE_CURRENT_BINARY_DIR}/src/driver_config.h ${CMAKE_CURRENT_SOURCE_DIR}/driver_config.h
```

Comment:
At some point we need to consider removing these 2 lines, we have a similar copy in `prepare-src`:
https://github.com/stackrox/collector/blob/495eca30a96996f52636427de7e698ba56f37a08/kernel-modules/build/prepare-src#L93-L95
https://github.com/stackrox/collector/blob/495eca30a96996f52636427de7e698ba56f37a08/kernel-modules/build/prepare-src#L99-L100